fs/xfstests.py

```diff
@@ -147,10 +158,10 @@ def tearDown(self):
         if os.path.exists(self.test_mnt):
             os.rmdir(self.test_mnt)
         if self.dev_type == 'loop':
-            for dev in self.loop_devices:
+            for dev in self.devices:
                 process.system('losetup -d %s' % dev, shell=True,
                                sudo=True, ignore_status=True)
-        self.part.unmount()
+            self.part.unmount()
```

Comment:
unmount is needed for disks also, since we need to mount them after creating filesystem